src/latched.rs

```diff
@@ -655,6 +656,31 @@ impl<
     }
 }
 
+impl<
+        const ROWS: usize,
+        const COLS: usize,
+        const NROWS: usize,
+        const BITS: u8,
+        const FRAME_COUNT: usize,
+    > FrameBufferUser<ROWS, COLS, NROWS, BITS, FRAME_COUNT>
+    for DmaFrameBuffer<ROWS, COLS, NROWS, BITS, FRAME_COUNT>
+{
+    #[inline]
+    fn erase(&mut self) {
+        self.erase();
+    }
+
+    #[inline]
+    fn format(&mut self) {
```

Comment:
`format()` is only used once during new() its not needed and I should probably make it non-public.
So I think we can not have it as part of this trait.